src/Phinx/Migration/Migration.template.php.dist

```diff
@@ -1,8 +1,9 @@
 <?php
+declare(strict_types=1);
```

Comment:
Are we losing a newline here?

Reply:
Yes. But it looks correct because `$namespaceDefinition` appears to always be empty. If it is possible to populate that variable, the code should also prefix and suffix a blank line around the namespace definition. Of course, it is not possible to add such logic to this template and [I don't know how the code surrounding namespaces actually works](https://github.com/cakephp/phinx/issues/1820) (if it does at all).

Reply:
This works fine as if `$namespaceDefinition` is defined, it bookends the string with `PHP_EOL`s